src/Sylius/Component/Addressing/Model/PostCode.php

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: mamazu
+ * Date: 01/02/18
+ * Time: 11:21
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Component\Addressing\Model;
+
+
+class PostCode implements PostCodeInterface
+{
+    /** @var int|null */
+    protected $id;
+
+    /** @var string */
+    protected $name;
+
+    /** @var string */
+    protected $postCode;
+
+    /** @var CountryInterface */
+    protected $country;
+
+    /** {@inheritdoc} */
+    public function getId(): ?int
+    {
+        return $this->id;
+    }
+
+    /** {@inheritdoc} */
+    public function setId(?int $id): void
+    {
+        $this->id = $id;
+    }
+
+    /** {@inheritdoc} */
+    public function getCode(): string
+    {
+        $countryCode = $this->country === null ? '  ' : $this->country->getCode();
+        return "{$countryCode}-{$this->postCode}";
```

Comment:
We don't use [fluent interfaces](https://ocramius.github.io/blog/fluent-interfaces-are-evil/) anymore

Reply:
I don't get why this should be a fluent interface. It is just a method that formats the post code.